pkg/api/upstreamconversions/conversions.go

```diff
@@ -0,0 +1,41 @@
+package upstreamconversions
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapiv1 "k8s.io/kubernetes/pkg/api/v1"
+	extensionsapi "k8s.io/kubernetes/pkg/apis/extensions"
+	extensionsv1beta1 "k8s.io/kubernetes/pkg/apis/extensions/v1beta1"
+	"k8s.io/kubernetes/pkg/conversion"
+	"k8s.io/kubernetes/pkg/runtime"
+)
+
+func AddToScheme(scheme *runtime.Scheme) {
+	addConversionFuncs(scheme)
+
+	// this alias allows for conversions from v1.RCs to internal RSs.  It's pretty weird.  I think I want a GroupVersioner
+	// that can specify the kind that it wants
+	scheme.AddKnownTypeWithName(extensionsapi.SchemeGroupVersion.WithKind("ReplicationController"), &extensionsapi.ReplicaSet{})
```

Comment:
@smarterclayton I'm pretty sure that I'm having to do this because https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/runtime/scheme.go#L515 returns a GVK based on a `GroupVersioner` (https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/pkg/runtime/codec.go#L221), which can't say, "in this GroupVersion, I want to be that GroupVersionKind".  The kind is being forced to the GVK I'm converting from, so I have to alias it.

I think we should fix the `GroupVersioner`, but I'm not even sure that group versioners made it upstream.